gui/default/syncthing/core/syncthingController.js

```diff
@@ -1789,13 +1791,30 @@ angular.module('syncthing.core')
         };
 
         $scope.addFolder = function () {
+            $('#folderLabel').change(function() {
+                var path = $('#folderPath').val();
+                $http.get(urlbase + '/db/path?path=' + encodeURIComponent(path))
+                .success(function (data) {
+                    if(data.ignore){
+                        $('#folder-ignores textarea').val(data.ignore);
```

Comment:
Remove the disabled attr here as well? We've loaded the ignore patterns, so we might as well let the user edit them before creating the folder

Reply:
Already at this level the user can modify the ignores.

Reply:
But we set the `disabled` attributes sometimes (when there is an error loading the folder), and we reuse the same modal for adding a folder here, so I think we still need to remove the `disabled` attribute, even if we think it's probably not set.